src/utils/setParameters.js

```diff
@@ -131,6 +131,10 @@ export default function setParameters (params) {
     dom.addClass(popup, params.customClass)
   }
 
+  if (params.customContainerClass && typeof params.customContainerClass === 'string') {
```

Comment:
`typeof params.customContainerClass === 'string'` <- unnecessary check, it's already in `dom.addClass`.